python/init.sls

```diff
@@ -28,6 +28,7 @@ pip:
       {% endif %}
     - reload_modules: True
 
+{% if not salt['pillar.get']('is_bootstrap') %}
```

Comment:
There's no `is_bootstrap` pillar; use `{% if salt['pillar.get']('fully_managed', True) %}` for now instead.